tests/autoyast/installation.pm

```diff
@@ -126,6 +126,20 @@ sub run {
                 send_key_until_needlematch 'create-partition-plans-finished', $cmd{ok};
                 next;
             }
+            if (match_has_tag('bsc#1055034') || match_has_tag('bsc#1054895')) {
+                record_soft_failure('bsc#1055034');
+                if (check_screen 'bsc#1054895', 0) {
+                    record_soft_failure('bsc#1054895');
+                }
+                send_key $cmd{ok};
+                next;
+            }
+            if (match_has_tag('bsc#1056356')) {
+                record_soft_failure('bsc#1056356');
```

Comment:
shouldn't the needle be already marked with the workaround property so that you don't need these record_soft_failure calls?